### ISSUE TITLE
[Bug][ExpansionPanel] Tabbing not working as expected

### ISSUE THREAD

**Issue by filipKovachev** [telerik/kendo-vue#665]
**Describe the bug**
The tab ordering seems broken on the expansion panel, when inspecting it has the first item as tabindex="0" and subsequent items as 1,2,3,4 on paper this should be fine but for some reason it's not tabbing through them, however if you click on the second item and tab through it will tab through them and at the end it will tab to the first item.

**To Reproduce**

Open this example: https://stackblitz.com/edit/sq7u3nt5-rywfergw?file=src%2Fmain.vue
Start tabbing and notice that you can't actually go further than the first item

**Expected behavior**
Tabbing should work, the same way it does for the React version:

https://stackblitz.com/run/?file=app%2Fapp.jsx

Reported in Ticket ID: 1677524
